Makefile

```diff
@@ -152,6 +158,7 @@ add-license-headers: $(GO_ADD_LICENSE)
 	
 .PHONY: kind-up
 kind-up:
+	@printf "\n\033[0;33m📌 NOTE: To target the newly created KinD cluster, please run the following command:\n\n    export KUBECONFIG=$(KUBECONFIG_PATH)\n\033[0m\n"
```

Comment:
When i run `make kind-up` then it prints the following message to the console:
```bash
📌 NOTE: To target the newly created KinD cluster, please run the following command:
    export KUBECONFIG=/Users/i062009/src/github.com/gardener/etcd-repos/etcd-druid/hack/e2e-test/infrastructure/kind/kubeconfig
```
I am not sure why are we using e2e test's kube-config?

Reply:
Default `KUBECONFIG_PATH` is `hack/e2e-test/infrastructure/kind/kubeconfig` used by [`Makefile`](https://github.com/gardener/etcd-druid/blob/master/Makefile#L24). So, it is pointing to e2e-test folder.

Reply:
ok i never understood what is the reason to checkin a kubeconfig. Especially when it is mentioned in `docs/development/local-e2e-tests.md`
```
Tip: Create a fresh [KinD](https://kind.sigs.k8s.io/) cluster or a similar one with a small footprint before executing the tests.
```